source/user-manual/reference/ossec-conf/vuln-detector.rst

```diff
@@ -32,6 +32,10 @@ Options
 +---------------------------+------------------------------------+
 | `index-status`_           | ``yes``, ``no``                    |
 +---------------------------+------------------------------------+
+| `offline-url`_            | ``<path>`` (starting with file://) |
+|                           | or ``<URL>``                       |
+|                           | (starting with http:// or https://)|
```

Comment:
I would remove the hyperlinks here. 

Here's an example I found within the documentation 

https://documentation.wazuh.com/current/user-manual/user-administration/single-sign-on/administrator/pingone.html
![image](https://github.com/wazuh/wazuh-documentation/assets/13010397/7653ea86-4603-4625-a3f8-61381b33fc32)

Reply:
Done. Thx!